### PR TITLE
Clarify iOS Carthage instructions, add warning about manual install

### PR DESCRIPTION
**Pull request by mspensieri**

### CODE REVIEW

File: pages/guide/native-ios-sdk.md [hunk 1]
```diff
@@ -36,32 +34,35 @@ Finally, install the pod
...
+
+```
+github "smooch/smooch-ios"
+```
+
+And finally, follow the [installation steps](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos) for iOS projects to link the Smooch dependency to your application.
+
+That's it! You're now ready to [initialize Smooch in your app](/guide/native-ios-sdk/#import-the-smooch-header-file).
+
+### Manual Installation
```
Comment (alavers) [inline pages/guide/native-ios-sdk.md]: Seems weird to have a manual installation section only to tell you not to. What if we just got rid of it?

Comment (alavers) [reply pages/guide/native-ios-sdk.md]: I looked at a few other popular cocoapod projects at random and their installation instructions mention cocoapods and/or carthage, and nothing else. /shrug

Comment (mspensieri) [reply pages/guide/native-ios-sdk.md]: > What if we just got rid of it?

I imagine people who think they know what they're doing will try it and get the error we're warning about and contact our support. This section is meant to discourage people from trying it

Comment (alavers) [reply pages/guide/native-ios-sdk.md]: 👍

**Pull request merged**